examples/fastai/conda.yaml

```diff
@@ -0,0 +1,11 @@
+channels:
```

Comment:
Can the name be put at the top and changes to:
name: fastai-example

Reply:
Also formatting for this file, 4 spaces :P

Reply:
The other conda.yaml and Project files are not using 4 spaces, but I can. We can do a second scoop over all exampes/* code 
and adhere to PEP8. One of things lacking is the example guidelines. Will work on that as part of the CON TRIBUTION.rst

Reply:
Looke likes we currently use 2 spaces for yaml files

Reply:
Yes, we are.

Reply:
I usually use 2 spaces

Reply:
@harupy @arjundc-db I have incorporated all your feedback. A couple of items:

1. It seems like the DCO test keeps on failing because I accepted one of [your commits](https://github.com/mlflow/mlflow/pull/3158/commits/cdb03094ab3ae5f6fbffa37d6c7f052dbbe7ff2f) that was not signed-off.

1. @harupy you had mentioned that we should probably enable github-actions for `pytest` examples and `lint`. How do I do that? Or is that something you can do.

Let me know if I missed any of your suggestions.